sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/CryptographyConstants.java

```diff
@@ -34,6 +43,23 @@
     static final String DECRYPT_UNENCRYPTED_BLOB = "Encryption client is being used but the blob metadata indicates "
         + "that it is not encrypted.";
 
+    // This is properties file's name.
+    private static final String AZURE_STORAGE_PROPERTIES = "azure-storage-blob.properties";
+
     private CryptographyConstants() {
     }
+//
+//    private static String getName() {
+//        UserAgentProperties properties = CoreUtils.getUserAgentProperties(AZURE_STORAGE_PROPERTIES);
+//        return properties.getName();
+//    }
+//
+//    private static String getVersion() {
+//        UserAgentProperties properties = CoreUtils.getUserAgentProperties(AZURE_STORAGE_PROPERTIES);
+//        return properties.getVersion();
+//    }
+
+    private static UserAgentProperties getProperties() {
+        return CoreUtils.getUserAgentProperties(AZURE_STORAGE_PROPERTIES);
+    }
```

Comment:
Does this need to be a separate method or could it be in-lined with the field instantiation?